docs/source/standard.md

```diff
@@ -255,39 +255,41 @@ The ability of the organisation to support users who want to browse the data ava
 
 ## 4. Information security
 
-The ability of the organisation to protect against the unauthorized use of information, especially electronic data and the measures taken to achieve this.
+A TRE conforming to the SATRE standard should enact broadly similar measures to protect against the unauthorised use of information, especially electronic data. These measures include vulnerability management of TRE infrastructure (whether physical or virtual/cloud-based), carrying out compliance checks and security tests of the TRE, common approaches to data encryption, and (where appropriate) physical security measures to prevent unauthorised access to the TRE .
```

Comment:
I think we probably want to keep the capability wording that "X is the ability of Y to do Z" although it's fine to expand on that as you've done here.